Packs/TheHiveProject/IncidentFields/incidentfield-Case_PAP.json

```
@@ -0,0 +1,36 @@
+{
+ "associatedToAll": false,
+ "associatedTypes": [
+  "Hive"
+ ],
+ "caseInsensitive": true,
+ "cliName": "casepap",
```

please prefix with thehive
```suggestion
 "cliName": "thehivecasepap",
```